src/fastly/include-util.js

```diff
@@ -13,6 +13,14 @@
 const fs = require('fs-extra');
 const path = require('path');
 
+function regex(content, filePath) {
+  return content.replace(/\{"regex:block.rgx"\}/g, () => {
+    const name = path.resolve(filePath);
+    const c = fs.readFileSync(name).toString().split('\n').join('');
```

Comment:
- why sync ?
- why not `readFile(name, 'utf-8')` (no toString() needed)

Reply:
I use sync because the rest of the code does! I can change this if you'd like! thanks for the tip about using 'utf-8'! :)